Procfile

```diff
@@ -1 +0,0 @@
-web: hypercorn -k trio --bind 0.0.0.0:5000 'lnbits.app:create_app()'
```

Comment:
Used when starting an instance on Heroku. But the command is wrong since we've switched to FastAPI. We could remove that and re-add if someone is interested in it.